lib/ble_zephyr.patch

```diff
@@ -0,0 +1,158 @@
+diff --git a/lib/stm32wb/hci/ble_bufsize.h
+index bf9ea29f..91dd8fef 100644
+--- a/lib/stm32wb/hci/ble_bufsize.h
++++ b/lib/stm32wb/hci/ble_bufsize.h
+@@ -8,9 +8,9 @@
+  * All rights reserved.</center></h2>
+  *
+- * This software component is licensed by ST under Ultimate Liberty license
+- * SLA0044, the "License"; You may not use this file except in compliance with
+- * the License. You may obtain a copy of the License at:
+- *                             www.st.com/SLA0044
```

Comment:
Why is this file required ?
In usual cube packages updates, we're able to report fixes from one version to the next w/o any .patch file

Reply:
Reworked (file removed)